CHANGELOG.md

```diff
@@ -11,6 +11,8 @@
 
 ### Fixed
 * Support mapping into `SyncSessionErrorCode` for "Compensating write" with error code 231. ([#1022](https://github.com/realm/realm-dart/pull/1022))
+* Previously it was difficult to setup a flutter project with realm, so that `flutter test` would work
```

Comment:
I can change it like that, but it will not be shorter :-)

Reply:
True, let me try again :)

Support `realm pub run install` command can be executed in Flutter projects which need to run Flutter unit tests and widget tests. This will make sure the required native binary is at the correct location.

```suggestion
Support install command in flutter projects that use unit and widget tests. 
```